molgenis-data-row-edit/src/main/frontend/src/repository/dataRowRepository.js

```diff
@@ -0,0 +1,96 @@
+import { EntityToFormMapper } from '@molgenis/molgenis-ui-form'
+import api from '@molgenis/molgenis-api-client'
+
+const entityMapperSettings = {
+  showNonVisibleAttributes: true
```

Comment:
I can not see the "string_hidden label" field. I expected it to be visible by this option? The save button is disabled as you showed me yesterday.

Reply:
i can see the 'xstring_hidden label' field , is this the field you mean of is there some other field with the id='string_hidden label'

Reply:
when i flip 'showNonVisibleAttributes' to false the 'xstring_hidden label' field is removed as expected